config/initializers/rspec_openapi.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'rspec/openapi'
+
+# Set request `headers` - generate parameters with headers for a request
+RSpec::OpenAPI.request_headers = %w[access-token uid client]
+
+# Set response `headers` - generate parameters with headers for a response
+RSpec::OpenAPI.response_headers = %w[access-token expiry token-type uid client]
```

Comment:
maybe we don't need to have these?
```suggestion
RSpec::OpenAPI.response_headers = %w[]
```
?

Reply:
I left these since they were present from the other doc config. I'm not really sure we need them